Makefile

```diff
@@ -12,6 +12,7 @@ clean:
 	rm -f ./build.log
 
 package-install:
+	npm audit --audit-level=moderate
```

Comment:
Adding this at the initial stage of the build, ensures fast failure on issues and smallest threat window with vulnerable or compromised packages.

Reply:
Wait, how is it able to scan dependencies and sub-dependencies before they are installed?

Reply:
Yeah I don't think npm audit will work if there's no package-lock file / node_modules

Reply:
See: https://ci.platform.aws.chdev.org/teams/team-tiny-rebels/pipelines/dissolution-web/jobs/pull-request-analysis/builds/274

Passed CI build with functional report showing found 70 low severity vulnerabilities in 1571 scanned packages and no >=moderate vulnerabilities.

Local testing shows anything >= to audit level fails build.